src/image.c

```diff
@@ -61,7 +33,7 @@ static _Bool img_palette_initialized;
 #define AMASK 0xff000000
 
 // image formats, tried in this order
-static const char *img_formats[] = { "tga", "png", "jpg", "wal", "pcx", NULL };
```

Comment:
Hm, again, I think i'd like to retain the ability to load .jpg, just in case. It's free.